### PR TITLE
Add mode on transaction type params

### PR DESCRIPTION
**Pull request by abe-phdev**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -9,6 +9,6 @@ def selectable(data)
   end
 
   def titleized_enum(types)
-    types.map { |x| [x[0].gsub(/_/, ' ').titleize, x[1]] }
+    types.map { |x| [x[0].gsub(/_/, ' ').titleize, x[0]] }
```
Comment (rusbal) [inline app/helpers/application_helper.rb:12]: Why the change here?
**The results:**
Before
`["Abc Dog", 1]`
After
`["Abc Dog", "abc_dog"]`

Comment (rusbal) [reply app/helpers/application_helper.rb:12]: Bugfix right?  If so, please continue.

Comment (abe-phdev) [reply app/helpers/application_helper.rb:12]: ["Abc Dog", 1]
It is showing that 1 is not a valid mode and I tried to change x[1] to x[0] to get the "Abc Dog" and it solves the issue.

Comment (rusbal) [reply app/helpers/application_helper.rb:12]: Ok good.

**Review by rusbal — APPROVED** [review]
Lgtm.
